ui/src/theme.js

```diff
@@ -369,6 +386,18 @@ const createTheme = isDarkTheme => {
           },
         },
       },
+      MuiDialogActions: {
+        root: {
+          '& button': {
+            color: isDarkTheme ? 'white' : '#000',
```

Comment:
Instead of using `white` and `#000`, I wonder if it would make more sense to use `THEME.PRIMARY_TEXT_DARK` and `THEME.PRIMARY_TEXT_LIGHT`.